lib/cylc/cfgspec/suite.py

```diff
@@ -298,6 +298,9 @@ def _coerce_final_cycletime( value, keys, args ):
                 'host'                        : vdr( vtype='string' ),
                 'owner'                       : vdr( vtype='string' ),
                 'suite definition directory'  : vdr( vtype='string' ),
+                'retrieve job log'            : vdr( vtype='boolean', default=False ),
+                'retrieve job log max size'   : vdr( vtype='string' ),
+                'retrieve job log delays'     : vdr( vtype='interval_minutes_list', default=[] ),
```

Comment:
These should probably be "retrieve job logs" (plural) ... plus corresponding change to documentation.